stdlib/2and3/cgitb.pyi

```diff
@@ -0,0 +1,33 @@
+
+from typing import Dict, Any, List, Tuple, Optional, Callable, Type, Union, IO, AnyStr, TypeVar
+from types import FrameType, TracebackType
+import sys
+
+
+_T = TypeVar("_T")
+_ExcInfo = Tuple[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]]
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _Path = Union[_T, PathLike[_T]]
+else:
+    _Path = Union[_T]
+
+
+def reset() -> str: ...
+def small(text: str) -> str: ...
+def strong(text: str) -> str: ...
+def grey(text: str) -> str: ...
+def lookup(name: str, frame: FrameType, locals: Dict[str, Any]) -> Tuple[Optional[str], Any]: ...
+def scanvars(reader: Callable[[], bytes], frame: FrameType, locals: Dict[str, Any]) -> List[Tuple[str, Optional[str], Any]]: ...
+def html(einfo: _ExcInfo, context: int = ...) -> str: ...
+def text(einfo: _ExcInfo, context: int = ...) -> str: ...
+
+class Hook:
+
+    def __init__(self, display: int = ..., logdir: Optional[_Path[AnyStr]] = ..., context: int = ..., file: Optional[IO[str]] = ..., format: str = ...) -> None: ...
+    def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
+    def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
+
+handler: Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], None]
```

Comment:
From the documentation it does not sound like this is meant to be overridden. So I think it makes more sense to use a straight `def` here in the stubs.